src/Simplex/Messaging/Server/CLI.hs

```diff
@@ -33,12 +35,10 @@ exitError :: String -> IO ()
 exitError msg = putStrLn msg >> exitFailure
 
 confirmOrExit :: String -> IO ()
-confirmOrExit s = do
-  putStrLn s
-  putStr "Continue (Y/n): "
-  hFlush stdout
-  ok <- getLine
-  when (ok /= "Y") exitFailure
+confirmOrExit s =
+  withPrompt (s <> "\nContinue (Y/n): ") $ do
+    ok <- getLine
+    when (ok /= "Y") exitFailure
```

Comment:
it's better to print something, e.g. "Aborting..." before `exitFailure` - it's not entirely clear if server files weren't deleted if you type "y" lowercase for example.. (it just does nothing). Or at least compare with it as well
```suggestion
    when (ok /= "Y" && ok /= "y") exitError "Aborting deletion"
```